inc/customizer/settings.php

```diff
@@ -53,6 +53,34 @@ function _customize_additional_scripts( $wp_customize ) {
 add_action( 'customize_register', '_customize_additional_scripts' );
 
 /**
+ * Register settings for site identity
+ * The section title_tagline is provided by Largo under the label "Site Identity"
+ */
+function _customize_title_tagline( $wp_customize ) {
+	// Register a setting
+	$wp_customize->add_setting(
+		'site_blurb',
+		array(
+			'default'           => '',
+			'sanitize_callback' => 'wp_filter_nohtml_kses',
```

Comment:
Some sites are currently using HTML in that field.

Examples taken from vagrant databases:

> The non-profit <strong>Connecticut Health Investigative Team</strong> is dedicated to producing original, responsible, in-depth journalism on issues of health and safety. <a href="/about/">Learn More&nbsp;&raquo;</a>

> Our mission is to promote the public good by pursuing in-depth, underreported and high-impact journalism which exposes injustice and explores its consequences. Our reporting will investigate the truth, hold those in power accountable, and seek solutions. <a href="/about/">Learn&nbsp;More&nbsp;&raquo;</a>

> The Global Investigative Journalism Network (GIJN) is composed of nonprofit investigative journalism organizations that produce stories, conduct training, provide resources, and encourage the creation of similar nonprofit groups. <a href="/about">Learn&nbsp;More&nbsp;&raquo;</a>

> A collaborative journalism initiative among public media, with roots in CO, WY, and ND. Funded by a grant from the Corporation for Public Broadcasting, our mission is to create a more informed public on energy issues by inspiring community conversations. <a href="http://insideenergy.org/support/">Learn more about how Inside Energy is funded. </a>

It looks like about 1 in 2 sites has configured this field; 1 in 6 uses HTML here.